.github/ISSUE_TEMPLATE/config.yml

```
@@ -0,0 +1,14 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Bug Report - Netdata Cloud
+    url: https://github.com/netdata/netdata-cloud/issues/new/choose
+    about: Create a report to help us improve our SaaS offering
```

Avoid using the term "SaaS" when talking about Netdata Cloud. It doesn't really fit into the SaaS model.

```suggestion
    about: Create a report to help us improve our web application
```